File: app/src/main/java/com/open/learncode/java/泛型/java/接口/ShowClass_1.java
package com.open.learncode.java.泛型.java.接口;

public class ShowClass_1 implements IShow<String> {
    public void show(String str) {
        System.out.println("泛型接口，实现类确定了类型 show:" + str);
    }
}

File: app/learncode-app/src/main/java/com/open/learncode/java/泛型/kotlin/接口/ShowClass_1.kt
package com.open.learncode.java.泛型.kotlin.接口

import com.open.learncode.java.泛型.java.接口.IShow

class ShowClass_1 : IShow<String> {
    override fun show(str: String) {
        println("泛型接口，实现类确定了类型 show:$str")
    }
}